### PR TITLE
#17: Add 'copy' button for each key and value in the key-values view

### PR DESCRIPTION
**Pull request by olena-syrotenko**
Implemented copying as an event, it will work if we click on the key-value row, so the "copy" icon is not needed as a functionality, but can be added as a functionality description element. Also, changed filling of the table depending on the selected folder.

### CODE REVIEW

File: app_view.py [hunk 1]
```diff
@@ -118,6 +121,12 @@ def populate_vault_node_data(self, v_node):
             if not self.folders.exists(new_item_id):
                 self.folders.insert(v_node.get_full_path(), tk.END, text=node.name, iid=new_item_id, open=False)
 
+    def fill_vault_data_table(self, node_path):
+        self.vault_data.delete(*self.vault_data.get_children())
+        data = self.viewmodel.get_node_data(node_path)
```
Comment (syrotenko) [inline app_view.py:126]: Do we really need to get data using vault?
Selected node already should contain key-values. Probably, it's better just to get currently selected node and fetch `kv` from it

File: app_view.py [hunk 2]
```diff
@@ -127,4 +136,9 @@ def _on_folders_node_open(self, event):
                 self.populate_vault_node_data(child)
 
     def _on_folder_selected(self, event):
-        print('NOT YET IMPLEMENTED')
+        node_path = self.folders.focus()
+        self.fill_vault_data_table(node_path)
+
+    def _on_copy(self, event):
+        item = self.vault_data.selection()[0]
+        values = self.vault_data.item(item, 'values')
```
Comment (syrotenko) [inline app_view.py:144]: It's better to implement separate coping for both key and value. If TKinter doesn't allow implement it in the fast way, leave the current implementation

**Review by syrotenko — CHANGES_REQUESTED** [review]

File: app_view.py (hunk 2 above)
Comment (olena-syrotenko) [reply app_view.py]: `Values` is a tuple that contains both key and value, so we are able to get them using the index. If nedded, I will add separate variables for key and value

File: app_view.py (hunk 1 above)
Comment (olena-syrotenko) [reply app_view.py:126]: Maybe this way will be better if we store this information in node finally. But when we each time access to vault it ensures that data are up to date

**Review by syrotenko — APPROVED** [review]

**Pull request merged**